shared/js/dataDriver.js

```diff
@@ -360,10 +376,19 @@ loop.DataDriver = function() {
      *  - {Mixed}  value     Stored value for the record.
      */
     _processRecord(key, data) {
-      let dispatchAction;
-      let dispatchExtra = {};
       // XXX akita bug 1274130: Validate data/key/timestamp before processing.
       let [, type, id] = key.match(/^([^!]+)!(.+)$/);
```

Comment:
The `key` should be validated although you could probably just do this by saving the result of `match` to check if it's `null` before doing the destructured assignment.